pkg/migrations/sql/preinstall/000-utils.sql

```diff
@@ -1,9 +1,24 @@
-CREATE OR REPLACE PROCEDURE execute_everywhere(command TEXT, transactional BOOLEAN = true)
+ --perms for schema will be addressed later;
+ CREATE SCHEMA IF NOT EXISTS SCHEMA_CATALOG;
```

Comment:
kinda sad we have to do this here, but I don't think there's a better way to do this...